api/deploy.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/stormkit-io/stormkit-cli/model"
+	"github.com/stormkit-io/stormkit-cli/stormkit"
+)
+
+// DeployByIDapi is the api string formatter for Sprintf(), first argument
+// app id, second argument deploy id
+const DeployByIDapi = "/app/%s/deploy/%s"
+
+// DeployByID calls the stormkit http api with the appID and the log id,
+// it returns a SingleDeploy struct that is the rappresentation of the
+// http response
+func DeployByID(appID, id string) (*model.SingleDeploy, error) {
```

Comment:
**suggestion** I would rename this to something like: `DeployLogsByID`.

Reply:
But this API pull the whole deploy right and for this command we are just using the logs, but maybe we will use again this func for fetch and use the whole deploy object, WDYT?

Reply:
Ah you're right, sorry about that! 👍

Reply:
No problem 😜